src/CommandLine/Program.cs

```diff
@@ -38,6 +38,44 @@ static int Main(string[] args)
 
             app.HelpOption(inherited: true);
 
+            void SubscribeTopicPerEventType(CommandLineApplication subscribeCommand)
+            {
+                subscribeCommand.Description = "Subscribes an endpoint to an event using topic-per-event topology.";
+                var name = subscribeCommand.Argument("name", "Name of the endpoint (required)").IsRequired();
+                var topicName = subscribeCommand.Argument("topic", "Topic name to subscribe. Unless configured otherwise on the publisher defaults to full name of the event (e.g. MyNamespace.MyMessage) (required)").IsRequired();
```

Comment:
I got a bit confused here because by looking at the description I put myself into the shoes of the user of the command line tool and the wordings says "defaults" which to me indicated the command argument is indeed optional